buildbot-buildmaster/buildbot/secrets/README.md

```diff
@@ -0,0 +1,7 @@
+This is the place for
+- htpasswd file
```

Comment:
Why is the `.htpasswd` file empty? If it is a template and must be filled, please provide information what needs to be added (and an example)

Reply:
Added example to .htpasswd and githubToken